doc/jit.md

```diff
@@ -0,0 +1,44 @@
+# Testarossa JIT (tr.jit) compiler
+
+Testarossa JIT compiler is a high performant dynamic compiler that powers openj9 - an enterprise-grade Java Virtual Machine (JVM). It's characteristic features are to compress the application code for improved runtime performance by exploiting the exact host CPU model and apply profiling on the runtime behavior of the generated code, feed it back to the optimizer to further refine it. The component provides a wide range of command line options to (i) trace the actions of the compilation and the compiled code, and (ii) control the compiler behavior.
```

Comment:
* 'high performance', or 'highly performant'
* 'OpenJ9', not 'openj9' 
* Talking in the abstract about the component is odd -- I'd instead simply talk about the 'compiler'

Reply:
thanks @mgaudet - made those changes.